folium/__init__.py

```diff
@@ -88,7 +88,6 @@
     'MacroElement',
     'Map',
     'Marker',
-    'MarkerCluster',
```

Comment:
My bad! (Probably a bad rebase :-/)

Do we want to remove `MarkerCluster` from this list?

Reply:
It's not imported in `folium/__init__.py`, instead it's available in `folium/plugins/__init__.py`. So it shouldn't be listed in the `__all__` of `folium/__init__.py` I think.

Reply:
I guess it was left here for historic reasons. I prefer to import from plugins like the rest.